apps/frontend/src/app/KeyMap/StatIcon.tsx

```diff
@@ -0,0 +1,83 @@
+import { allElementsWithPhy, ElementKeyWithPhy } from "@genshin-optimizer/consts"

```

Comment:
Is there a reason we should put this in KeyMap and not Components, where it used to be?

Reply:
It will eventually be powered by the keymap lookup in wr2,